b2sdk/api.py

```diff
@@ -440,34 +442,54 @@ def create_key(
             valid_duration_seconds=valid_duration_seconds,
             bucket_id=bucket_id,
             name_prefix=name_prefix
+            # As of 17.06.2021 the server does not accept an 'options' argument
```

Comment:
that's because it's a read-only property, You can use it to see if the bucket is S3-capable or not (and I guess legalhold-capable nowadays?)